src/state/slices/opportunitiesSlice/selectors/stakingSelectors.ts

```diff
@@ -54,15 +55,15 @@ export const selectStakingIds = createDeepEqualOutputSelector(
 
 export const selectUserStakingIds = createDeepEqualOutputSelector(
   selectEnabledWalletAccountIds,
-  (state: ReduxState) => state.opportunities.userStaking.ids,
+  opportunities.selectors.selectUserStakingIds,
```

Comment:
Unable to comment line, but remove L48 unused

Reply:
bb3b521cab